src/main/java/assets/game.js

```diff
@@ -81,7 +97,7 @@ function markHits(board, elementId, surrenderText) {
 
 function redrawGrid() {
 
-    if(spaceLaserEngaged == false && game.opponentsBoard.remainingShips < 3 && !isSetup) {
+    if(spaceLaserEngaged == false && game.opponentsBoard.remainingShips < 4 && !isSetup) {
```

Comment:
Thanks for fixing this, the space laser needing to engage after 1 sunk ship (and not 2) was an oversight on my part.